lib/reek/cli/options.rb

```diff
@@ -71,6 +72,7 @@ def set_alternative_formatter_options
         end
       end
 
+      # :reek:TooManyStatements: : { max_statements: 6 }
```

Comment:
I know it’s super nitpicky, but can we do away with the double colons? They creep me out. :)